pokeapp/src/components/Layout.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Layout = ({ Navigation, FilterBar, SideBar, DigitalCardBinder }) => {
+  return (
+    <div className="main-container">
+      {Navigation}
+      {FilterBar}
+      {SideBar}
+      {DigitalCardBinder}
+    </div>
```

Comment:
Nice and simple with plenty of room for expansion.  Good choice on using slot pattern (especially if you want to offer your user a selection of layouts!).